controllers/configurationpolicy_controller.go

```diff
@@ -2556,6 +2558,30 @@ func handleSingleKey(
 			mergedValue.(map[string]interface{}), existingValue.(map[string]interface{}))
 	}
 
+	if key == "stringData" && existingObj.GetKind() == "Secret" {
```

Comment:
I am curious about what happens and what is our expected result when a user adds the string `stingDate`, in other words, not encoded data. for example,
```
- complianceType: musthave
          objectDefinition:
            apiVersion: v1
            kind: Secret
            metadata:
              name: htpasswd-secret
              namespace: openshift-config
            stringData:
              htpasswd: "password"
```

Reply:
somewhere throw an error?